docs/releasenotes/index.rst

```diff
@@ -9,6 +9,14 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
+v1.0.1
+======
+
+Upgrading is strongly recommended for all users of 1.0.0.
```

Comment:
This is generally a question for the whole of the docs - do we refer to v1 as v1.0 or v1.0.0 - we use both on this page. 

Also, I would recommend always using the "v"